fleet_vehicle_configuration/models/fleet_configuration.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2022 Camptocamp (https://www.camptocamp.com).
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class FleetVehicleConfiguration(models.Model):
+    _name = "fleet.vehicle.configuration"
+    _description = "Manage vehicle configuration"
+
+    vehicle_id = fields.Many2one("fleet.vehicle")
```

Comment:
```suggestion
    vehicle_id = fields.Many2one("fleet.vehicle", required=True, ondelete="cascade")
```
?

Reply:
I can't set it as required as it works like "order_id" field on sale_order_line

Reply:
What do you mean? `order_id` is required in sale.order.line 😅 

The question is: does it make sense to have `fleet.vehicle.configuration` without an actual `vehicle`? If so, what do they mean?